HPXMLtoOpenStudio/measure.rb

```diff
@@ -2470,15 +2482,27 @@ def self.add_heat_pump(runner, model, building, weather)
 
       load_frac_heat = heat_pump_values[:fraction_heat_load_served]
       if @total_frac_remaining_heat_load_served > 0
-        sequential_load_frac_heat = load_frac_heat / @total_frac_remaining_heat_load_served # Fraction of remaining load served by this system
+        if (load_frac_heat - @total_frac_remaining_heat_load_served).abs <= 0.010001
+          # Last equipment to handle all the remaining load (within 0.01 tolerance)
+          load_frac_heat = @total_frac_remaining_heat_load_served
+          sequential_load_frac_heat = 1.0 # Fraction of remaining load served by this system
+        else
+          sequential_load_frac_heat = load_frac_heat / @total_frac_remaining_heat_load_served # Fraction of remaining load served by this system
+        end
       else
         sequential_load_frac_heat = 0.0
       end
       @total_frac_remaining_heat_load_served -= load_frac_heat
 
       load_frac_cool = heat_pump_values[:fraction_cool_load_served]
       if @total_frac_remaining_cool_load_served > 0
-        sequential_load_frac_cool = load_frac_cool / @total_frac_remaining_cool_load_served # Fraction of remaining load served by this system
+        if (load_frac_cool - @total_frac_remaining_cool_load_served).abs <= 0.010001
+          # Last equipment to handle all the remaining load (within 0.01 tolerance)
+          load_frac_cool = @total_frac_remaining_cool_load_served
+          sequential_load_frac_cool = 1.0 # Fraction of remaining load served by this system
+        else
+          sequential_load_frac_cool = load_frac_cool / @total_frac_remaining_cool_load_served # Fraction of remaining load served by this system
+        end
```

Comment:
Seems like similar code added to 4 different places. Can we create a method that is called by all 4 places?

Reply:
Sounds like a good idea